packages/kit/src/runtime/server/page/actions.js

```diff
@@ -214,7 +214,7 @@ async function call_action(event, actions) {
 
 	if (!is_form_content_type(event.request)) {
 		throw new Error(
-			`Actions expect form-encoded data (received ${event.request.headers.get('content-type')}`
+			`Actions expect form-encoded data (received ${event.request.headers.get('content-type')})`
```

Comment:
Another alternative would be to remove the parentheses. I'll leave this as a suggestion to see if the other maintainers have any preference 
```suggestion
			`Actions expect form-encoded data, but received ${event.request.headers.get('content-type')}`
```